### ISSUE TITLE
Buggy behavior all of a sudden with docsets updates

### ISSUE THREAD

**Issue by stefanos82** [zealdocs/zeal#1569]
I'm using version `0.7.0-3` under GNU / Linux Debian testing 64-bit, fully updated and I have the following issue: I checked for updates and saw there were a couple of them, like in the following screenshot.

![image](https://github.com/zealdocs/zeal/assets/1777466/ee544910-0909-43cf-b351-ebe70d950c63)

After I updated them all, on reopening Zeal and checked docsets updates again, the same result as screenshot above remains the same.

Here's the whole process logged via terminal:

```
stefanos@debian:~ $ zeal 
zeal.core.applicationsingleton: Singleton ID: 6IXruwgvJZ_xjXaNbIAGjB1qnj6ALNagcafapCbHh1g
zeal.core.applicationsingleton: Starting as a primary instance. (PID: 46798)
qt.webenginecontext: 

GL Type: desktop
Surface Type: OpenGL
Surface Profile: CompatibilityProfile
Surface Version: 4.6
QSG RHI Backend: OpenGL
Using Supported QSG Backend: yes
Using Software Dynamic GL: no
Using Multithreaded OpenGL: yes

Init Parameters:
  *  application-name Zeal 
  *  browser-subprocess-path /usr/lib/qt6/libexec/QtWebEngineProcess 
  *  create-default-gl-context  
  *  disable-features ConsolidatedMovementXY,InstalledApp,BackgroundFetch,WebOTP,WebPayments,WebUSB,PictureInPicture 
  *  disable-setuid-sandbox  
  *  disable-speech-api  
  *  enable-features NetworkServiceInProcess,TracingServiceInProcess 
  *  enable-threaded-compositing  
  *  in-process-gpu  
  *  use-gl desktop 

zeal.core.httpserver: Listening on http://127.0.0.1:54079...
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Boost.docset/Contents/Resources/Documents' to '/boost'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Bootstrap_5.docset/Contents/Resources/Documents' to '/bootstrap_5'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/C++.docset/Contents/Resources/Documents' to '/c__'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/C.docset/Contents/Resources/Documents' to '/c'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/CMake.docset/Contents/Resources/Documents' to '/cmake'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset/Contents/Resources/Documents' to '/css'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Emmet.docset/Contents/Resources/Documents' to '/emmet'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Erlang.docset/Contents/Resources/Documents' to '/erlang'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset/Contents/Resources/Documents' to '/html'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset/Contents/Resources/Documents' to '/javascript'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/jQuery.docset/Contents/Resources/Documents' to '/jquery'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/jQuery_UI.docset/Contents/Resources/Documents' to '/jquery_ui'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset/Contents/Resources/Documents' to '/laravel'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Lua_5.4.docset/Contents/Resources/Documents' to '/lua_5_4'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Nginx.docset/Contents/Resources/Documents' to '/nginx'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Perl.docset/Contents/Resources/Documents' to '/perl'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/PHP.docset/Contents/Resources/Documents' to '/php'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/PostgreSQL.docset/Contents/Resources/Documents' to '/postgresql'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/React.docset/Contents/Resources/Documents' to '/react'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Ruby_on_Rails_7.docset/Contents/Resources/Documents' to '/ruby_on_rails_7'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/SQLite.docset/Contents/Resources/Documents' to '/sqlite'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/WordPress.docset/Contents/Resources/Documents' to '/wordpress'.
zeal.core.httpserver: Unmounted prefix 'Laravel' ('/laravel').
zeal.core.filemanager: Removing '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset'...
zeal.core.filemanager: Renamed '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset' to '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset.1697383614268.deleteme'.
zeal.core.filemanager: Removed '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset.1697383614268.deleteme'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/Laravel.docset/Contents/Resources/Documents' to '/laravel'.
zeal.core.httpserver: Unmounted prefix 'HTML' ('/html').
zeal.core.filemanager: Removing '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset'...
zeal.core.filemanager: Renamed '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset' to '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset.1697383679448.deleteme'.
zeal.core.filemanager: Removed '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset.1697383679448.deleteme'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/HTML.docset/Contents/Resources/Documents' to '/html'.
zeal.core.httpserver: Unmounted prefix 'JavaScript' ('/javascript').
zeal.core.filemanager: Removing '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset'...
zeal.core.filemanager: Renamed '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset' to '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset.1697383793626.deleteme'.
zeal.core.filemanager: Removed '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset.1697383793626.deleteme'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/JavaScript.docset/Contents/Resources/Documents' to '/javascript'.
zeal.core.httpserver: Unmounted prefix 'CSS' ('/css').
zeal.core.filemanager: Removing '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset'...
zeal.core.filemanager: Renamed '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset' to '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset.1697383811678.deleteme'.
zeal.core.filemanager: Removed '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset.1697383811678.deleteme'.
zeal.core.httpserver: Mounted '/home/stefanos/.local/share/Zeal/Zeal/docsets/CSS.docset/Contents/Resources/Documents' to '/css'.
```

**Comment by mabo85** [comment]
I have the same issue on my private machine with Archlinux and Zeal 0.7.0.

The `meta.json` file of those misbehaving docsets are missing the version number _and_ the revision number.

On another machine with Zeal 0.6.1 the version number and revision are present in the `meta.json` file of the docset. There the issue of this behaviour is absent.

**Comment by stefanos82** [comment]
Ah that's why when I hover over CSS, the tooltip says `Version: r0`; very good find @mabo85 :+1: 

Is there anything we need to do or is it something that has to do with those who provide the docset package?

**Comment by wdscxsj** [comment]
Duplicate of #1559. Just wait for v0.7.1.

**Comment by trollixx** [comment]
As mentioned above, this is fixed in v0.7.1.
